storagenode/storagenodedb/pieceexpiration.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedb
+
+import (
+	"context"
+	"time"
+
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storagenode/pieces"
+)
+
+type pieceExpirationDB struct {
+	*InfoDB
+}
+
+// PieceExpirationDB returns database for storing piece expiration data
+func (db *DB) PieceExpirationDB() pieces.PieceExpirationDB { return db.info.PieceExpirationDB() }
+
+// PieceExpirationDB returns database for storing piece expiration data
+func (db *InfoDB) PieceExpirationDB() pieces.PieceExpirationDB { return &db.pieceExpirationDB }
```

Comment:
Does this function more properly live in infodb.go?

Reply:
Possibly, yes. But this is following the pattern used for all the other sub-DBs inside InfoDB. See bandwidthdb.go, consoledb.go, orders.go, pieceinfo.go, usedserials.go, and vouchers.go.

Reply:
> this is following the pattern

Fair enough. I prefer a pattern I don't like to no pattern at all. :)